.all-contributorsrc

```diff
@@ -222,6 +222,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "sawyerh",
+      "name": "sawyerh",
+      "avatar_url": "https://avatars.githubusercontent.com/u/371943?v=4",
+      "profile": "https://github.com/sawyerh",
+      "contributions": [
+        "code"
+      ]
```

Comment:
Looks like this was manually edited, since I don't see a corresponding generated change to the README. 

We mostly use the all contributors bot to add contributions, which I'll put a documentation PR up for.

I'll also tell all-contributors to add you for code! For now, please reset these changes and I'll approve!

Reply:
Updated!